### ISSUE TITLE
AssemblyShipmentSerializer super class mismatch issue

### ISSUE THREAD

**Issue by RhysStansfield** [spree-contrib/spree-product-assembly#94]
Running into a slightly annoying issue (2-3-stable) whereby the Spree::Wombat::AssemblyShipmentSerializer is regularly throwing a superclass mismatch error after changes are made in our app in development mode - issue comes from this line of code:

https://github.com/spree/spree-product-assembly/blob/2-3-stable/lib/spree_product_assembly/engine.rb#L14 

Changing it to simply require it fixes the problem - I assume there was a reason to be load-ing it though in the first place, testing or some-such?

I also note that in the current master version of this Gem you've removed it entirely (not an option for us at the moment unfortunately as we can't move up to master spree without doing some serious hackery).

Long story short - can I open a PR removing that load jazz from 2-3-stable or will it cause devastation in ways I've not considered?  Or other options/ideas?

Cheers!

Rhys

**Comment by proletarian** [comment]
I'm getting exactly the same issue. Any movement on this issue?

**Comment by zavan** [comment]
Same error here.
